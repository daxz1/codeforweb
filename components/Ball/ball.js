import React from 'react';

/**
 *
 * @param x
 * @param y
 * @param fill
 * @returns {*}
 * @constructor
 */
export const Ball = ({ x, y, fill }) => (<circle cx={x} cy={y} r={5} fill={fill}/>);