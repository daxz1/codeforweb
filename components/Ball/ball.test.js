/*global describe, expect */
import React from 'react';
import { Ball } from './ball';
import renderer from 'react-test-renderer';

describe('BALL TEST SUITE', ()=> {
  it('Ball Renders Correctly', () => {
    const tree = renderer.create(<Ball x="0" y="0" fill="#fffff" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});