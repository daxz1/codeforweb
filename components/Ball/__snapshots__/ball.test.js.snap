// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BALL TEST SUITE Ball Renders Correctly 1`] = `
<circle
  cx="0"
  cy="0"
  fill="#fffff"
  r={5}
/>
`;
